ghost/audience-feedback/lib/AudienceFeedbackService.js

```diff
@@ -1,7 +1,21 @@
+const urlUtils = require('../../core/core/shared/url-utils');
+
 class AudienceFeedbackService {
-    buildLink() {
-        // todo
-        return new URL('https://example.com');
+    /**
+     * @param {string} uuid
+     * @param {string} postId
+     * @param {0 | 1} score
+     */
+    buildLink(uuid, postId, score) {
+        const siteUrl = urlUtils.getSiteUrl();
+        const params = new URLSearchParams({
+            action: 'feedback',
+            post: postId,
+            uuid,
+            score
+        });
+
+        return new URL(`${siteUrl}?${params.toString()}`);
```

Comment:
This is a little convoluted, I think it would read better as something like:
```js
const url = new URL(siteUrl);
url.searchParams.set('action', 'feedback');
// etc...

return url;
```

Reply:
Fixed :)